src/Form/BecomeAdherentType.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class BecomeAdherentType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->remove('firstName')
+            ->remove('lastName')
+            ->remove('emailAddress')
+            ->remove('position')
+        ;
+    }
+
+    public function getParent()
+    {
+        return MembershipRequestType::class;
```

Comment:
Please rename this one instead, something like `UserProfileType` or `User...whatever`

Reply:
Ok but it makes more sense to call it `AdherentType` as it's about modifying an `Adherent`.